.gitlab-ci.yml

```diff
@@ -6,10 +6,9 @@ stages:
 
 validate:
   stage: ssa-validate
-  image: python:3.7
+  image: chrsep/alpine-java-python
   before_script:
     - pip3 install -r requirements.txt
   script:
     - python3 bin/validate.py -p . -v
```

Comment:
I think Jose wanted you to remove the non-SSA validate since that's done elsewhere.